ccdproc/core.py

```diff
@@ -1740,6 +1741,94 @@ def _sigma_mask(baseline, one_sigma_value, lower_sigma, upper_sigma):
     return mask
 
 
+def bitmask2mask(bitmask, ignore_bits=0, flip_bits=None):
```

Comment:
@mcara if your preferred name involves bitfields_to_boolean_mask, then I'd be okay with going with that.